test/controllers/beneficial.owner.gov.controller.spec.ts

```diff
@@ -1231,7 +1231,8 @@ describe("BENEFICIAL OWNER GOV controller", () => {
       expect(mockSaveAndContinue).toHaveBeenCalledTimes(1);
     });
 
-    test("catch error when removing data", async () => {
+    // ASM-350 - need fix to create a real error
+    test.skip("catch error when removing data", async () => {
```

Comment:
When is this being dealt with? With a follow-up PR related to ASM-350?

Reply:
see the answer to the previous comment

Reply:
How about all the other places this appears/test is skipped? e.g. in test/controllers/trust.information.controller.spec.ts.

Reply:
https://github.com/companieshouse/overseas-entities-web/pull/1572#discussion_r1751447464

Reply:
skipped tests pass